doc/building-running.md

```diff
@@ -0,0 +1,63 @@
+**Validated: 2020/03/26**
+
+# Building Cardano Node
+
+There are a number of ways the Cardano Node can be built and run, but the following document
+details the Nix and Ubuntu/Debian methods. The Nix method is probably more robust and foolproof
+than installing on Ubuntu/Debian.
+
+
+### Building under Nix
+
+The [Nix Package Manager][nix] can be installed on most Linux distributions by downloading and
+running the installation script:
+```
+curl https://nixos.org/nix/install > install-nix.sh
+./install-nix.sh
+```
+and following the directions.
+
+To improve build speed, it is possible to set up a binary cache maintained by IOHK (**this is
+optional**):
+```
+sudo mkdir -p /etc/nix
+cat <<EOF | sudo tee /etc/nix/nix.conf
+substituters = https://cache.nixos.org https://cache.nixos.org/ https://iohk.cachix.org https://hydra.iohk.io https://cache.nixos.org/
+trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= iohk.cachix.org-1:DpRUyj7h7V830dp/i6Nti+NEO2/nhblbov/8MW7Rqoo= hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
```

Comment:
`cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=` occurs twice on this line